### PR TITLE
[SPARK-38825][SQL][TEST] Add a test to cover parquet notIn filter

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?
Currently we don't have a test for parquet `notIn` filter, so add a test for this

### Why are the changes needed?
to make tests more complete

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
new test

### CODE REVIEW

**Comment by huaxingao** [comment]
cc @dongjoon-hyun @viirya @sunchao

File: sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala [hunk 1]
```diff
@@ -1901,6 +1901,27 @@ abstract class ParquetFilterSuite extends QueryTest with ParquetTest with Shared
       }
     }
   }
+
+  test("in and notIn filters") {
```
Comment (dongjoon-hyun) [inline sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala:1905]: Please add a test prefix, `SPARK-38825: `.

**Review by dongjoon-hyun — APPROVED** [review]

**Review by viirya — APPROVED** [review]

**Review by sunchao — APPROVED** [review]
LGTM with the nit @dongjoon-hyun mentioned.

**Comment by huaxingao** [comment]
I have tested this manually. I will merge this without pending CI.

**Comment by huaxingao** [comment]
Merged to master/3.3. Thank you all!

**Pull request closed without merging**